src/box/txn_limbo.h

```diff
@@ -232,6 +232,11 @@ struct txn_limbo {
 	 * can't be any inconsistencies.
 	 */
 	bool do_validate;
+	/**
+	 * The time that the latest successfully confirmed entry waited for
+	 * quorum.
+	 */
+	double confirm_lag;
```

Comment:
Just successfully? Some entries are deleted due to a rollback (even when we delete limbo timeout rollback, the promotes still will be able to rollback txns). The function `txn_limbo_remove()` is used only by successful commit, but its name doesn't suggest that anyhow. I suggest to rename it or even better - place the update right into `txn_limbo_read_confirm()`.

Reply:
> Just successfully?

Yes, I don't think rollbacks are of interest, and #9918 explicitly mentions successfully confirmed entries.

> I suggest to rename it or even better - place the update right into txn_limbo_read_confirm().

Then I will have to patch two places in `txn_limbo_read_confirm` that remove async and sync transactions (not sure it will be enough to update the confirm lag only for synchronous transactions). So let's rename `txn_limbo_remove` and `txn_limbo_pop` to `txn_limbo_pop_confirmed` and `txn_limbo_pop_rolled_back`.